### PR TITLE
Improve export dialog UI

### LINKED ISSUE #517: Pixels field title in export window not aligned to the left
Right click on photo > Other Actions > Export...
Screenshot from 2019-07-30 23 09 28
It's not capitalized and doesn't have a colon. The pixels field itself might have to be thought out, as it's the same size as the selection fields. I know this is another UI element taken from Shotwell, so thought I'd bring it up if anyone felt it needed to be redesigned. 

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/77993400-pixels-field-title-in-export-window-not-aligned-to-the-left?utm_campaign=plugin&utm_content=tracker%2F60550306&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F60550306&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by fabiozaramella**
Fixes #517 

Old:
Old
New:
Screenshot from 2019-08-04 12 30 59@2x

### CODE REVIEW

**Comment by cassidyjames** [comment]
Definitely an improvement. I wonder: while we're here, would it make sense to switch the quality combo to a scale, and the size entry to a spinbutton?

**Comment by fabiozaramella** [comment]
@cassidyjames I already thought about switching to a spinbutton, but i didn't changed it because it requires a range of values and i wasn't sure about the proper number to put it (the current entry accept a maximum of 6 digits anyway so...). Should i update this PR or make a different one?

File: src/Dialogs/ExportDialog.vala [hunk 1]
```diff
@@ -94,13 +93,15 @@ public class ExportDialog : Gtk.Dialog {
...
+        pixels_entry = new Gtk.SpinButton.with_range (1, 999999, 1);
+        pixels_entry.set_max_length (6);
+        pixels_entry.set_value (current_scale);
+        pixels_entry.set_digits (0);
 
-        Gtk.Label pixels_label = new Gtk.Label.with_mnemonic (_("_pixels"));
-        pixels_label.mnemonic_widget = pixels_entry;
+        Gtk.Label size_label = new Gtk.Label.with_mnemonic (_("_Size in pixels:"));
+        size_label.halign = Gtk.Align.END;
+        quality_label.use_underline = true;
```
Comment (danrabbit) [inline src/Dialogs/ExportDialog.vala]: Is this a copy/paste mistake or is this property just in the wrong spot :)

File: src/Dialogs/ExportDialog.vala [hunk 2]
```diff
@@ -139,9 +141,11 @@ public class ExportDialog : Gtk.Dialog {
         constraint_combo.changed.connect (on_constraint_changed);
         format_combo.changed.connect (on_format_changed);
         pixels_entry.changed.connect (on_pixels_changed);
-        pixels_entry.insert_text.connect (on_pixels_insert_text);
         pixels_entry.activate.connect (() => {
-            response (Gtk.ResponseType.OK);
+            if ((pixels_entry.get_text_length () > 0) && (int.parse (pixels_entry.get_text ()) > 0))
```
Comment (danrabbit) [inline src/Dialogs/ExportDialog.vala]: Can we use brackets for if/else please

File: src/Dialogs/ExportDialog.vala (hunk 1 above)
Comment (fabiozaramella) [reply src/Dialogs/ExportDialog.vala]: Oops, copy/past mistake

File: src/Dialogs/ExportDialog.vala (hunk 2 above)
Comment (fabiozaramella) [reply src/Dialogs/ExportDialog.vala]: done :+1:

**Pull request merged**